Cargo.toml

```diff
@@ -36,6 +36,7 @@ rocket = { default-features = false, optional = true, version = "0.5.0-rc.3" }
 serde = { default-features = false, optional = true, version = "1.0" }
 serde_json = { default-features = false, optional = true, version = "1.0" }
 tokio-postgres = { default-features = false, optional = true, version = "0.7" }
+wasm-bindgen = { default-features = false, optional = true, version = "0.2" }
```

Comment:
I'm just looking through how some other libraries introduce this and I think it needs to be done as a target as opposed to a simple feature. It could technically _still_ be a feature however it'd need to be a no-op for std/core.

e.g. 
```toml
[target.'cfg(target_arch = "wasm32")'.dependencies]
wasm-bindgen = { default-features = false, version = "0.2" }
```

There may be a requirement to also _exclude_ some other libraries that aren't supported on wasm of course. However that'd require some investigation (e.g. what features are supported in combination with wasm)